coconut.sh

```diff
@@ -0,0 +1,45 @@
+package: coconut
+version: "%(commit_hash)s"
+tag:  master
+build_requires:
+  - golang
+  - protobuf
+source: https://github.com/AliceO2Group/Control
+incremental_recipe: |
+  make WHAT="coconut" all
+  mkdir -p $INSTALLROOT/bin
+  cp bin/* $INSTALLROOT/bin
+  mkdir -p $INSTALLROOT/etc/modulefiles && rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
+---
+#!/bin/bash -e
+
+GOPATH=$PWD
+PATH=$GOPATH/bin:$PATH
+BUILD=$GOPATH/src/github.com/AliceO2Group/Control
+mkdir -p $BUILD
+rsync -a --delete $SOURCEDIR/ $BUILD/
+cd $BUILD
+make WHAT="coconut" all
+mkdir -p $INSTALLROOT/bin
+cp bin/* $INSTALLROOT/bin
+cd -
+
+mkdir -p etc/modulefiles
+cat > etc/modulefiles/$PKGNAME <<EoF
+#%Module1.0
+proc ModulesHelp { } {
+  global version
+  puts stderr "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+}
+set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
+module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+# Dependencies
+module load BASE/1.0
+
+# Our environment
+set coconut_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
```

Comment:
The convention (even if I see it has been violated in other places) is to have the package name UPPERCASE, so - even if this variable won't go in the user's environment - please use `COCONUT_ROOT`.